app/helpers/decidim/vocdoni/votes_helper.rb

```diff
@@ -12,6 +12,22 @@ def ordered_answers(question)
       def more_information?(answer)
         translated_attribute(answer.description).present?
       end
+
+      def votes_left_message(votes_left)
+        scope = "decidim.vocdoni.votes.new"
+        max_votes = Decidim::Vocdoni.votes_overwrite_max
+
+        message_key, css_class = case votes_left
+                                 when (2..max_votes)
+                                   %w(can_vote_again secondary)
+                                 when 1
+                                   %w(can_vote_one_more_time warning)
+                                 when 0
+                                   %w(no_more_votes_left alert)
+                                 end
```

Comment:
```suggestion
        message_key, css_class = case votes_left
                                 when 1
                                   %w(can_vote_one_more_time warning)
                                 when 0
                                   %w(no_more_votes_left alert)
                                 else
                                   %w(can_vote_again secondary)
                                 end
```

I'm thinking that the current implementation might fail if you decide to reconfigure the plugin changing the "maxOverride" in the module configuration. Something like this should cover all the cases no?